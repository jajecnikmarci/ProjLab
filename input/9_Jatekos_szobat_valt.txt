starttest
add room room1 3
add room room2 3
add room room3 3
add door room1 room2 door1
add door room1 room3 door2
add person room1 Student student1
add person room3 Professor professor1
gotoroom student1 room2
gotoroom professor1 room1
printstate room1
printstate room2
printstate room3
printstate student1
printstate professor1
endtest