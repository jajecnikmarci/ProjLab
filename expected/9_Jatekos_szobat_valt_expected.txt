room1
	- capacity: 3
	- people: 
		professor1
	- doors: 
		door1
		door2
	- items: 
	- poisonEffects: 
	- stunEffects: 
	- lastCleaning: null
room2
	- capacity: 3
	- people: 
		student1
	- doors: 
		door1
	- items: 
	- poisonEffects: 
	- stunEffects: 
	- lastCleaning: null
room3
	- capacity: 3
	- people: 
	- doors: 
		door2
	- items: 
	- poisonEffects: 
	- stunEffects: 
	- lastCleaning: null
student1
	- location: room2
	- stunned: false
	- inventory: 
	- killImmunities: 
	- poisonImmunities: 
	- souls: 3
professor1
	- location: room1
	- stunned: false
	- Inventory: 
	- poisonImmunities: 
